motoman_sia50d_support/test/launch_test.xml

```diff
@@ -0,0 +1,59 @@
+<!--
+  launch_test.xml - ROSlaunch tests
+
+  Usage Instructions:
+
+  1. Add the following to your CMakeLists.txt:
+    
+    find_package(roslaunch)
+    roslaunch_add_file_check(test/launch_test.xml)
+
+  2. Create a test directory under your package
+  3. Add the "launch_text.xml" file and fill out the test below.  Use the
+     following conventions:
+    a. Encapsulate each launch file test in it's own namespace.  By
+       convention the namespace should have the same name as the launch
+       file (minus ".launch" extension)
+    b. Create tests for each possible combination of parameters.  Utilize
+       sub-namespaces under the main namespace.
+
+  Notes:
+
+  1. XML extension is used in order to avoid beinging included
+  in roslaunch auto-complete.
+
+  2. Group tags with namespaces are used to avoid node name collisions when
+  testing multpile launch files
+-->
+<launch>
+	<arg name="req_arg" value="default"/>
+  <arg name="dx100" value="dx100"/>
+
+  <group ns="load_sia50d">
+	  <include file="$(find motoman_sia50d_support)/launch/load_sia50d.launch"/>
+  </group>
+
+  <group ns="test_sia50d">
+	  <include file="$(find motoman_sia50d_support)/launch/test_sia50d.launch"/>
+  </group>
+
+
+  <group ns="robot_interface_streaming_sia50d">
+    <group ns="dx100" >
+	    <include file="$(find motoman_sia50d_support)/launch/robot_interface_streaming_sia50d.launch">
+		    <arg name="robot_ip"   value="$(arg req_arg)" />
+        <arg name="controller" value="$(arg dx100)"/>
```

Comment:
CI is likely failing because of this line.

You're passing the `controller` arg, but it's not actually something you can set: it already has a `value` (namely: `dx100`).